draft-ietf-dnsop-svcb-https.md

```diff
@@ -867,7 +867,30 @@ compliant recursive resolvers.  When SvcDomainName is ".", server operators
 SHOULD NOT include these hints, because they are unlikely to convey any
 performance benefit.
 
+## "mandatory" {#mandatory}
 
+In a ServiceForm RR, a SvcParamKey is considered "essential" if the RR will not
+function correctly for clients that ignore this SvcParamKey.  Each SVCB
+protocol mapping SHOULD specify a default set of essential keys.  The
+SvcParamKey "mandatory" is used to indicate the set of essential keys for this
```

Comment:
Is there a reason to switch between the words "essential" and "mandatory"?  Using both is somewhat distracting.

Reply:
I’d prefer to stick with “mandatory” and “default mandatory set”.  Agreed that the mix is a bit confusing.

Reply:
I chose this wording because "mandatory", in my view, is extremely confusing: "mandatory" sounds like domain owners are required to publish or use the "mandatory" keys, and clients are expected to enforce that they are present (neither of which is true).  These keys aren't "mandatory"; they're "mandatory-to-understand-if-present".  "Non-ignorable".

If you can think of a clearer way to express this I would appreciate a suggestion!

Reply:
I changed this to "mandatory"/"automatically mandatory".